facefusion/ffmpeg.py

```diff
@@ -146,7 +146,20 @@ def restore_audio(target_path : str, output_path : str, output_video_fps : Fps)
 	if isinstance(trim_frame_end, int):
 		end_time = trim_frame_end / output_video_fps
 		commands.extend([ '-to', str(end_time) ])
-	commands.extend([ '-i', target_path, '-c:v', 'copy', '-c:a', state_manager.get_item('output_audio_encoder'), '-map', '0:v:0', '-map', '1:a:0', '-shortest', '-y', output_path ])
+
+	audio_encoder = state_manager.get_item('output_audio_encoder') or 'aac'
+
+	commands.extend([
```

Comment:
please keep it on one line like the original

Reply:
I fix the keep one line code like original.